config/config.go

```diff
@@ -0,0 +1,229 @@
+package config
+
+/*
+   This is a package that wraps the viper, a package designed to handle config
+   files, for the BCDA app.
+
+   Assumptions:
+   1. The configuration file is a env file
+   2. The configuration file, once it is made available to the application,
+   will stay immutable during the uptime of the application (exception is test)
+*/
+
+import (
+	"os"
+	"testing"
+    "time"
+
+	"github.com/spf13/viper"
+)
+
+// Global variable made available to any other package importing this package
+var envVars viper.Viper
+const testmode bool = false
+
+// Implementing a state machine tracking how things are going in this package
+const (
+	configgood    uint8 = 0
+	configbad     uint8 = 1
+	noconfigfound uint8 = 2
+)
+// State is a public variable for diagnostic purposes.
+var State uint8 = configgood // if config found and loaded, value not changed
```

Comment:
I'm not seeing any usages of State outside of this package, do you think we can make it private?